test/test-network/systemd-networkd-tests.py

```diff
@@ -2801,6 +2801,50 @@ def test_activation_policy(self):
             with self.subTest(test=test):
                 self._test_activation_policy(test)
 
+    def _test_activation_policy_required_for_online(self, policy, required):
+        self.setUp()
+        conffile = '25-activation-policy.network'
+        units = ['11-dummy.netdev', '12-dummy.netdev', '12-dummy.network', conffile]
```

Comment:
Ah, `12-dummy.network` is not listed in `units` array at the beginning of `NetworkdNetworkTests`.

Reply:
aha! thanks, corrected in the latest push